astroid/constraint/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
+
+"""Representations of logical constraints on values used during inference.
+"""
+
+from .constraint import get_constraints
+
+__all__ = ("get_constraints",)
```

Comment:
Just saw the comment from @DanielNoord https://github.com/PyCQA/astroid/pull/1189#discussion_r834192353

IMO it's fine to leave everything in one module for now. If it get's too big, we can always go back and split it but for 140 lines, it just adds unnecessary complexity.

Reply:
I recommended this because of the many "moving" commits in astroid that interfere with git blame. It's a little premature, but it would avoid such churn later on.

But if both of you think it should stay in one file, then please go ahead. And sorry for the churn I caused then.

Reply:
The git blame issue is an inconvenience, true. However we shouldn't overcomplicate things now just in anticipation of what could be. We're far way from any necessary split up, can add at least another 2000 lines 😄

Reply:
Then @david-yz-liu please revert the change. As I said, sorry for the inconvenience. Your work on this PR is much appreciated as it is one of the longer request features of `astroid` and `pylint`.